remote/gitea/gitea.go

```diff
@@ -139,8 +139,8 @@ func (c *client) Login(res http.ResponseWriter, req *http.Request) (*model.User,
 		return nil, nil
 	}
 
-	// Create Client with Basic Auth
-	client, err := c.newClientBasicAuth(username, password)
+	// Create Client with Basic Auth or Token
```

Comment:
How about move if there is a password here?

Reply:
where? should I put a `move`? 

sorry - I think you have to explain your idea :/

Reply:
where? should I put a `move`? 

sorry - I think you have to explain your idea :/ @lunny